File: php/2011-final-value-of-variable-after-performing-operations.php
class Solution {

    /**
    * @param String[] $operations
    * @return Integer
    */
    function finalValueAfterOperations($operations) {
        $x = 0;
        $n = count($operations);
        for ($i = 0; $i < $n; $i++) {
            if ($operations[$i][1] === '+') {
                $x++;
            } else {
                $x--;
            }
        }
        return $x;
    }
}

File: golang/2011-final-value-of-variable-after-performing-operations.go
package golang

func finalValueAfterOperations(operations []string) int {
    x, n := 0, len(operations)
    for i := 0; i < n; i++ {
        if operations[i][1] == '+' {
            x++
        } else {
            x--
        }
    }
    return x
}